### ISSUE TITLE
Rog Ally - Turbo mode use Balanced power plans 

### ISSUE THREAD

**Issue by Rocktaze** [seerge/g-helper#2182]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

Instead of using the Turbo power plan, turbo mode use Balanced power plan. 

### How to reproduce the bug?

![3W0EGlw 1](https://github.com/seerge/g-helper/assets/76700478/de37f68a-78c6-49d4-a5bb-f7299d21f249)
![mIoSWo5 1](https://github.com/seerge/g-helper/assets/76700478/03c45407-2b4c-41ed-9a83-69bcb16da848)

### Logs

No logs 

### Device and Model

Asus og Ally

### Armoury Crate

Installed

### Asus Services

9

### Version

1.54

### OS

W11

**Comment by seerge** [comment]
@Rocktaze it's absolutely intentional, app relies on default balanced power plan and sets windows power modes. Please [check readme ](https://github.com/seerge/g-helper?tab=readme-ov-file#rocket-performance-modes)

**Comment by Rocktaze** [comment]
Testing with 3dmark timespy CPU score and frame time are somehow more consistent and higher when i manually choose Turbo power plans before starting the benchmark VS leaving it at Balanced.

So there must be difference somewhere in the balanced vs turbo power plan.

**Comment by seerge** [comment]
@Rocktaze Turbo and others are power plans coming with with AC :) and G-Helper doesn't rely on AC installation.

If you want, you can manually assign any power plan to any mode in a config as explained here https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode :)

**Comment by Rocktaze** [comment]
Can you ELI5 how would i go about Using Turbo Windows power plan with the Turbo mode in ghelper?
I'm not sure i understand your instruction.
